styles.css

```diff
@@ -330,7 +329,7 @@ i.fa-github {
   color: black;
 }
 
-i.fa-github:hover {
+a i.fa-github:hover {
```

Comment:
Hi Reinaldo, it's perfect i can see the use of tag a 👍

Reply:
👍